app/build.gradle

```diff
@@ -9,12 +9,12 @@ plugins {
 }
 
 android {
-    compileSdk 31
+    compileSdk 33
 
     defaultConfig {
         applicationId "com.pec_acm.moviedroid"
         minSdk 21
-        targetSdk 31
+        targetSdk 33
```

Comment:
Please remove the changes to `compileSdk` and `targetSdk`

Reply:
@ken1000minus7 if these are removed the libraries will have to be reverted as they have a min version that is higher than 31. This compiles just fine and is most likely the best direction to go with keeping your project up to date. Is there a problem ?

Reply:
Oh I see, then maybe we can update them as well🤔

Reply:
Interesting 😯, thanks for the advice😄